### PR TITLE
btUniversalConstraint

### PR DESCRIPTION
**Pull request by Smoozilla**
hope this is good, works in my project for what I need, thanks for the repo

Edit: Meant to not include the builds, but ah well there they are.

### CODE REVIEW

File: make.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def build():
   if not wasm:
     args += ' -s WASM=0 -s AGGRESSIVE_VARIABLE_ELIMINATION=1 -s ELIMINATE_DUPLICATE_FUNCTIONS=1 -s SINGLE_FILE=1 -s LEGACY_VM_SUPPORT=1'
   else:
-    args += ''' -s WASM=1 -s BINARYEN_IGNORE_IMPLICIT_TRAPS=1 -s BINARYEN_TRAP_MODE="clamp"'''
```
Comment (willeastcott) [inline make.py]: What does this change do?

**Comment by Smoozilla** [comment]
> see https://bugreports.qt.io/browse/QTBUG-76771 for make.py update

@willeastcott

File: make.py (hunk 1 above)
Comment (Smoozilla) [reply make.py]: see https://bugreports.qt.io/browse/QTBUG-76771 for make.py update

Comment (Smoozilla) [reply make.py]: more specifically, it allows building wasm with latest emscripten

**Comment by kripken** [comment]
Thanks @Smoozilla ! idl and makefile changes look good.

The builds however seem to be without closure? If so then they are bigger. Please either remove them, or build with closure.

**Comment by Smoozilla** [comment]
Honestly the more I've used 6dofspring and 6dofspring2, I've realised how useless this joint can be, thanks for your time!

**Pull request closed without merging**